### ISSUE TITLE
[Bug] Error during response generation: Error: Failed to generate response

### ISSUE THREAD

**Issue by alicat22** [waywysw/TalOS-Reborn#3]
**Describe the bug**
Failed to generate response on Discord after sending a message.

**To Reproduce**
1. Connect bot
2. Register room
3. `/swapchar` and add character
4. Send message
5. See error

**Expected behavior**
Bot should send a message in response.

**Desktop (please complete the following information):**
OS: Windows 11
Browser: Chrome
Version: 120.0.6099.217

**Additional context**
Refreshing page, disconnecting/reconnecting bot, restarting server, logging out and back in, Disconnecting API and reconnecting API all do not fix the issue. Deleting the room and re-registering also does not fix. Manually creating room on the UI does not fix, either.
```js
[1] Logged in as TalOS-Reborn#7320!
[1] Started refreshing application (/) commands.
[1] Successfully reloaded application (/) commands.
[1] The following commands were set: [
[1]   'clear',
[1]   'swapchar',
[1]   'registerroom',
[1]   'naigenerate',
[1]   'alias',
[1]   'sys',
[1]   'greeting',
[1]   'clearwebhooks'
[1] ]
[1] {
[1]   defaultConnection: '',
[1]   defaultSettings: '1705538965019',
[1]   admins: [ '1' ],
[1]   enableCaptioning: false,
[1]   enableEmbedding: false,
[1]   enableQuestionAnswering: false,
[1]   enableZeroShotClassification: false,
[1]   enableYesNoMaybe: false,
[1]   defaultDiffusionConnection: '',
[1]   jwtSecret: '0f0eaeeee2e186346cde58d7e9f5bcfa7245e9f7b22314f9308877bb60f1d25133046e16b9e3151f472c57f8a8518421bfac127b17f937e8a9b3248c3e604424'
[1] }
[1] Error during response generation: Error: Failed to generate response
[1]     at RoomPipeline.<anonymous> (file:///H:/AI/TalOS-Reborn/TalOS-Reborn/dist/services/discordBot/roomPipeline.js:257:31)
[1]     at Generator.next (<anonymous>)
[1]     at fulfilled (H:\AI\TalOS-Reborn\TalOS-Reborn\node_modules\tslib\tslib.js:166:62)
[1] Error during response generation: Error: Failed to generate response
[1]     at RoomPipeline.<anonymous> (file:///H:/AI/TalOS-Reborn/TalOS-Reborn/dist/services/discordBot/roomPipeline.js:257:31)
[1]     at Generator.next (<anonymous>)
[1]     at fulfilled (H:\AI\TalOS-Reborn\TalOS-Reborn\node_modules\tslib\tslib.js:166:62)
[1] {
[1]   defaultConnection: '',
[1]   defaultSettings: '1705538965019',
[1]   admins: [ '1' ],
[1]   enableCaptioning: false,
[1]   enableEmbedding: false,
[1]   enableQuestionAnswering: false,
[1]   enableZeroShotClassification: false,
[1]   enableYesNoMaybe: false,
[1]   defaultDiffusionConnection: '',
[1]   jwtSecret: '0f0eaeeee2e186346cde58d7e9f5bcfa7245e9f7b22314f9308877bb60f1d25133046e16b9e3151f472c57f8a8518421bfac127b17f937e8a9b3248c3e604424'
[1] }
[1] {
[1]   defaultConnection: '',
[1]   defaultSettings: '1705538965019',
[1]   admins: [ '1' ],
[1]   enableCaptioning: false,
[1]   enableEmbedding: false,
[1]   enableQuestionAnswering: false,
[1]   enableZeroShotClassification: false,
[1]   enableYesNoMaybe: false,
[1]   defaultDiffusionConnection: '',
[1]   jwtSecret: '0f0eaeeee2e186346cde58d7e9f5bcfa7245e9f7b22314f9308877bb60f1d25133046e16b9e3151f472c57f8a8518421bfac127b17f937e8a9b3248c3e604424'
[1] }
[1] {
[1]   defaultConnection: '',
[1]   defaultSettings: '1705538965019',
[1]   admins: [ '1' ],
[1]   enableCaptioning: false,
[1]   enableEmbedding: false,
[1]   enableQuestionAnswering: false,
[1]   enableZeroShotClassification: false,
[1]   enableYesNoMaybe: false,
[1]   defaultDiffusionConnection: '',
[1]   jwtSecret: '0f0eaeeee2e186346cde58d7e9f5bcfa7245e9f7b22314f9308877bb60f1d25133046e16b9e3151f472c57f8a8518421bfac127b17f937e8a9b3248c3e604424'
[1] }
[1] Error during response generation: Error: Failed to generate response
[1]     at RoomPipeline.<anonymous> (file:///H:/AI/TalOS-Reborn/TalOS-Reborn/dist/services/discordBot/roomPipeline.js:257:31)
[1]     at Generator.next (<anonymous>)
[1]     at fulfilled (H:\AI\TalOS-Reborn\TalOS-Reborn\node_modules\tslib\tslib.js:166:62)
[1] Error during response generation: Error: Failed to generate response
[1]     at RoomPipeline.<anonymous> (file:///H:/AI/TalOS-Reborn/TalOS-Reborn/dist/services/discordBot/roomPipeline.js:257:31)
[1]     at Generator.next (<anonymous>)
[1]     at fulfilled (H:\AI\TalOS-Reborn\TalOS-Reborn\node_modules\tslib\tslib.js:166:62)
[1] Error during message processing: Error: Failed to generate response
[1]     at RoomPipeline.<anonymous> (file:///H:/AI/TalOS-Reborn/TalOS-Reborn/dist/services/discordBot/roomPipeline.js:276:23)
[1]     at Generator.next (<anonymous>)
[1]     at fulfilled (H:\AI\TalOS-Reborn\TalOS-Reborn\node_modules\tslib\tslib.js:166:62)
```

**Comment by alicat22** [comment]
```js
}
[1] Logged in as TalOS-Reborn#7320!
[1] Started refreshing application (/) commands.
[1] Successfully reloaded application (/) commands.
[1] The following commands were set: [
[1]   'clear',
[1]   'swapchar',
[1]   'registerroom',
[1]   'naigenerate',
[1]   'alias',
[1]   'sys',
[1]   'greeting',
[1]   'clearwebhooks'
[1] ]
[1] {
[1]   defaultConnection: '',
[1]   defaultSettings: '1705538965019',
[1]   admins: [ '1' ],
[1]   enableCaptioning: false,
[1]   enableEmbedding: false,
[1]   enableQuestionAnswering: false,
[1]   enableZeroShotClassification: false,
[1]   enableYesNoMaybe: false,
[1]   defaultDiffusionConnection: '',
[1]   jwtSecret: '0f0eaeeee2e186346cde58d7e9f5bcfa7245e9f7b22314f9308877bb60f1d25133046e16b9e3151f472c57f8a8518421bfac127b17f937e8a9b3248c3e604424'
[1] }
[1] Error during response generation: Error: Failed to generate response
[1]     at RoomPipeline.<anonymous> (file:///H:/AI/TalOS-Reborn/TalOS-Reborn/dist/services/discordBot/roomPipeline.js:257:31)
[1]     at Generator.next (<anonymous>)
[1]     at fulfilled (H:\AI\TalOS-Reborn\TalOS-Reborn\node_modules\tslib\tslib.js:166:62)
[1] Error during response generation: Error: Failed to generate response
[1]     at RoomPipeline.<anonymous> (file:///H:/AI/TalOS-Reborn/TalOS-Reborn/dist/services/discordBot/roomPipeline.js:257:31)
[1]     at Generator.next (<anonymous>)
[1]     at fulfilled (H:\AI\TalOS-Reborn\TalOS-Reborn\node_modules\tslib\tslib.js:166:62)
[1] {
[1]   defaultConnection: '',
[1]   defaultSettings: '1705538965019',
[1]   admins: [ '1' ],
[1]   enableCaptioning: false,
[1]   enableEmbedding: false,
[1]   enableQuestionAnswering: false,
[1]   enableZeroShotClassification: false,
[1]   enableYesNoMaybe: false,
[1]   defaultDiffusionConnection: '',
[1]   jwtSecret: '0f0eaeeee2e186346cde58d7e9f5bcfa7245e9f7b22314f9308877bb60f1d25133046e16b9e3151f472c57f8a8518421bfac127b17f937e8a9b3248c3e604424'
[1] }
[1] {
[1]   defaultConnection: '',
[1]   defaultSettings: '1705538965019',
[1]   admins: [ '1' ],
[1]   enableCaptioning: false,
[1]   enableEmbedding: false,
[1]   enableQuestionAnswering: false,
[1]   enableZeroShotClassification: false,
[1]   enableYesNoMaybe: false,
[1]   defaultDiffusionConnection: '',
[1]   jwtSecret: '0f0eaeeee2e186346cde58d7e9f5bcfa7245e9f7b22314f9308877bb60f1d25133046e16b9e3151f472c57f8a8518421bfac127b17f937e8a9b3248c3e604424'
[1] }
[1] {
[1]   defaultConnection: '',
[1]   defaultSettings: '1705538965019',
[1]   admins: [ '1' ],
[1]   enableCaptioning: false,
[1]   enableEmbedding: false,
[1]   enableQuestionAnswering: false,
[1]   enableZeroShotClassification: false,
[1]   enableYesNoMaybe: false,
[1]   defaultDiffusionConnection: '',
[1]   jwtSecret: '0f0eaeeee2e186346cde58d7e9f5bcfa7245e9f7b22314f9308877bb60f1d25133046e16b9e3151f472c57f8a8518421bfac127b17f937e8a9b3248c3e604424'
[1] }
[1] Error during response generation: Error: Failed to generate response
[1]     at RoomPipeline.<anonymous> (file:///H:/AI/TalOS-Reborn/TalOS-Reborn/dist/services/discordBot/roomPipeline.js:257:31)
[1]     at Generator.next (<anonymous>)
[1]     at fulfilled (H:\AI\TalOS-Reborn\TalOS-Reborn\node_modules\tslib\tslib.js:166:62)
[1] Error during response generation: Error: Failed to generate response
[1]     at RoomPipeline.<anonymous> (file:///H:/AI/TalOS-Reborn/TalOS-Reborn/dist/services/discordBot/roomPipeline.js:257:31)
[1]     at Generator.next (<anonymous>)
[1]     at fulfilled (H:\AI\TalOS-Reborn\TalOS-Reborn\node_modules\tslib\tslib.js:166:62)
[1] Error during message processing: Error: Failed to generate response
[1]     at RoomPipeline.<anonymous> (file:///H:/AI/TalOS-Reborn/TalOS-Reborn/dist/services/discordBot/roomPipeline.js:276:23)
[1]     at Generator.next (<anonymous>)
[1]     at fulfilled (H:\AI\TalOS-Reborn\TalOS-Reborn\node_modules\tslib\tslib.js:166:62)
[1] true
[1] true
```

**Comment by alicat22** [comment]
Fixed with -> 2c001031254afa07f3ff6c0b5190514ede0a9e58
